nslsii/__init__.py

```diff
@@ -547,11 +547,96 @@ def migrate_metadata():
     new_md.update(old_md)
 
 
-def subscribe_kafka_publisher(RE, beamline_name, bootstrap_servers, producer_config):
+def subscribe_kafka_publisher(RE, publisher_queue, kafka_publisher):
```

Comment:
Slightly concerned about the API change.

Reply:
This should have been named to indicate it's not for public consumption. I have made that change.
Also I am confident this function is used nowhere else at present.